packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/shippingMethod.css

```diff
@@ -0,0 +1,9 @@
+.textContent {
+    padding: 25px;
+    padding-top: 50px;
+    font-weight: bold;
+}
+
+.proceedButton {
+    padding: 25px;
```

Comment:
We should be using `rem` units. Not super important as this'll be modified when we implement these components, but wanted to point it out.

Reply:
Done 👍